applications/nrf_desktop/configuration/nrf54l15dk_nrf54l15_cpuapp/images/mcuboot/prj.conf

```diff
@@ -30,6 +27,12 @@ CONFIG_UART_CONSOLE=n
 CONFIG_PRINTK=n
 CONFIG_USE_SEGGER_RTT=n
 
+CONFIG_PSA_USE_CRACEN_CIPHER_DRIVER=n
+CONFIG_PSA_USE_CRACEN_AEAD_DRIVER=n
+CONFIG_PSA_USE_CRACEN_HASH_DRIVER=n
```

Comment:
Ah, OK the driver only supports sha256, so it is useless with sha512 requirements for ed25519

Reply:
Is the SHA512 needed for pure ed25519?

Reply:
OK, so the comment around the HASH driver is misleading because it does not list the sha512.
The SHA512 is needed internally for pure, so it may not require driver.

Reply:
A am now building app with mcuboot gradually removing the kconfigs listed above and checking whether it still works.

Reply:
So OK, HASH needs to stay with non-pure